pcl_ros/include/sixriver/gpu/filters/impl/voxel_grid.hpp

```diff
@@ -335,6 +353,12 @@ sixriver::VoxelGrid<PointT>::applyFilter (PointCloud &output)
             int idx = ijk0 * divb_mul_[0] + ijk1 * divb_mul_[1] + ijk2 * divb_mul_[2];
             index_vector.push_back (cloud_point_index_idx (static_cast<unsigned int> (idx), *it));
         }
+
+        // Log WARN message if negative points detected
+        if (detect_negative_points_ && (number_of_negative_points > negative_point_number_threshold_))
+        {
+            ROS_WARN_THROTTLE(3.0, "Detecting %u point(s) below height threshold", number_of_negative_points);
```

Comment:
If we're trying to find the max that won't false alarm, throttling will only show us the first high reading every 3 seconds, but we'll still not be sure about the true highest. How often do you see this log message in e.g. a movement loop? Could we get away with not throttling it?

Reply:
Yeah, I am fine with removing the throttle. On the other hand I also don't want to spam the log because the code will be executed at 30 hz, so if the edge case gets hit, we will log a lot of messages.
This is the test I ran yesterday, if we set the number threshold to 0 and the robot is static, it will occasionally split out log with 16-30 points below -0.1m. For movement loop, ~150 points was the highest I saw yesterday with around 20 loops.
https://app.logdna.com/b2e069b3c4/logs/view?apps=&hosts=link&tags=rhqStaging&q=below%20height%20threshold